appserver/distributions/glassfish/pom.xml

```diff
@@ -65,6 +65,10 @@
             <type>zip</type>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.jline</groupId>
+            <artifactId>jline</artifactId>
+        </dependency>
```

Comment:
Please move this to https://github.com/eclipse-ee4j/glassfish/blob/master/appserver/featuresets/web/pom.xml so that it's also in GlassFish web profile distribution.